packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -13,6 +13,10 @@ export interface ISearchBoxState {
   id?: string;
 }
 
+const DefaultClearButtonProps: IButtonProps = {
```

Comment:
@aditima this is still a small nit i'd like changed. if we ever add a required prop to IButtonProps, this will throw an error. Otherwise, this is fine.